web_client/src/types.ts

```diff
@@ -1,8 +1,13 @@
-/* eslint-disable  @typescript-eslint/no-explicit-any *//* eslint camelcase: "off" */
+/* eslint-disable  @typescript-eslint/no-explicit-any */
+/* eslint camelcase: "off" */
 /* eslint no-unused-vars: "off" */
 /* eslint no-shadow: "off" */
 import type { WorkerPool } from 'itk/WorkerPool';
 
+// TODO: Throughout we have snake case properties, these
+// could be converted to camel case using:
+// https://morioh.com/p/8e8b33c25ea1
+
```

Comment:
We use snake case in the type file because that is how the keys are written when they are received from the backend, therefore request responses have snake case keys